packages/cli/lib/lib/webpack/webpack-base-config.js

```diff
@@ -22,6 +23,24 @@ function resolveDep(dep, cwd) {
 	return dep;
 }
 
+function findAllNodeModules(startDir) {
```

Comment:
this is necessary, because it would otherwise not resolve `node_modules` above the current folder, which is something I ran into when trying to use `preact-cli` in a project of mine with workspaces